layouts/partials/ecosystem/registry/entry.html

```diff
@@ -187,7 +191,7 @@ <h6><i class="fa-solid fa-forward"></i> Quick Install</h6>
                 <div class="fw-bold">
                     {{ . }}
                 </div>
-                <small>License</small>
+                <small data-bs-toggle="tooltip" title="This is the type of license for the project.">License</small>
```

Comment:
Provide some more details here, e.g. that there are "Commercial" licenses and end users have to check with the provider what that means, and that there are open source licenses, etc. You could also refer to the help page I suggested above and we provide some of the most common licenses there, what "Commercial" means and where people can find more infos about licenses, e.g. https://opensource.org/licenses